data/categories.txt

```diff
@@ -4795,7 +4795,7 @@ sk:3Toalety|wc
 mr:शौचालय
 
 amenity-university
-en:4University|U+1F393
+en:4University|U+1F393|2Uni
```

Comment:
2 means that after entering "Un", the app will show "Uni" as _first suggest_ result. Not good, imho, Uni is very specific term.

Can leave Uni without 2 prefix, but I have doubts about it.

Reply:
Uni is a commonly used acronym. I agree that digits should be removed.